docs/UserWallet.rst

```diff
@@ -112,23 +136,34 @@ Please note that this information is different than your current balance that di
 
 |image12|
 
+When you click on the plus icon next to the total rewarded, a drawer with all rewards details is displayed.
+
+|image18|
+
+You can see more information about a reward transaction by clicking in the corresponding line. The information about the number of gamification points earned and kudos recieved during the period of reward.
+
+|image19|
 
 Tansactions details
 ~~~~~~~~~~~~~~~~~~~~
 
-The amount of the last transaction is displayed in the wallet interface 
+The amount of the last transaction is displayed in the wallet interface.
 
 |image13|
 
-When you click on the plus icon next to the last transaction information, a drawer with all transactions is displayed 
+When you click on the plus icon next to the last transaction information, a drawer with all transactions is displayed.
 
 |image14|
 
 To see a transaction details (its status, label, message, amount, the sender and recipient adresses, the contact name and address, the transaction fee and hash) just click on the pull down arrow from the transaction list. 
 
 |image15|
 
+In the case of a failed transaction, a warning is displayed in the transaction list and the transaction details. The arrow and transaction amount are displayed in red color so that you can distingish them.
```

Comment:
```suggestion
In case of a failed transaction, a warning is displayed in the transaction list and the transaction details. The arrow and transaction amount are displayed in red color so that you can distingish them.
```